README-dev.md

```diff
@@ -70,7 +70,7 @@ $ yarn build
 It's easy to setup a new environment with docker compose :
 
 ```sh
-docker-compose up
+docker compose up
```

Comment:
Il faut utiliser `docker compose` à la place de `docker-compose` pour la gestion des secrets : https://stackoverflow.com/questions/72280771/how-to-use-secrets-when-building-docker-compose-locally/72280952#72280952